WTWSMS/history/characters/kurdish.txt

```diff
@@ -13,7 +13,7 @@
 	}
 }
 159547 = {
-	name="Balax�" #Balash
+	name="Balaxš" #Balash
```

Comment:
I don't know what happened here and in two characters under this, probably encoding changed.

Reply:
Yes, encoding was changed, now i fixed it.